apps/src/templates/teacherNavigation/lessonMaterials/LessonResources.tsx

```diff
@@ -92,11 +92,22 @@ const LessonResources: React.FC<LessonResourcesProps> = ({
     );
   };
 
+  const renderNoResourcesRow = () => {
```

Comment:
This can be moved outside of the functional component as a const and then it will never get re-rendered